src/hooks/index.ts

```diff
@@ -13,6 +13,7 @@ export {
 // TODO: Have to properly remove these as we're no longer using them internally.
 export { useThemeProps, usePropsWithComponentTheme } from './useThemeProps';
 // TODO: instead export only this.
+export * from '../hooks/useHasResponsiveProps';
```

Comment:
replace `*` with the intended name.

Reply:
Thank you for your feedback. I fixed it in the recent commit.